sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/JWT.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.springsdk.annotations;
+
+import kalix.JwtFieldOptions;
+
+import java.lang.annotation.*;
+
+import static kalix.JwtMethodOptions.JwtMethodMode;
+
+public @interface JWT {
+
+  @Target({ElementType.METHOD, ElementType.FIELD})
+  @Retention(RetentionPolicy.RUNTIME)
+  @Documented
+  @interface JwtMethodOptions {
```

Comment:
we can probably remove this extra level and only have `@JWT` annotation.

Reply:
done

Reply:
removed